app/views/404.njk

```diff
@@ -0,0 +1,12 @@
+{% extends "./layout.njk" %}
+
+{% block pageTitle %}
+  Page not found - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Page not found</h1>
+    <p class="govuk-body">Sorry, we could not find the page you were looking for.</p>
```

Comment:
This content should be put in the translation file, although I guess as the content is not finalised you might be holding off on that until we have the content

Reply:
Good spot - we will tackle this once we have the content.